### DISCUSSION TITLE
Buy Crazyflie 2.1 online

### DISCUSSION THREAD

**Discussion by Muzzamil-Ali-Hassan** [bitcraze/discussions#1607] [General]
I am a student currently pursuing my master’s degree in electronic engineering at Bursa Uludağ University in Turkey. I had read many scientific papers about nan-drones applications that used your product Crazyflie 2.x. This made me get deeper into your product and I became interested in this field and I want to do my final thesis using your product Crazyflie 2.x. If I want to buy Crazyflie online I ask how long it can take to reach my address in Türkiye?

**Comment by Bananazooka** [comment]
Hi, I'll close this issue since we're already talking by email.
